pinterest/teletraan
# Skip removing implicit deploy group on existing capacity update

Problem: 
---
Modification to existing group capacity overwrites env <-> group mapping entirely with passed in payload. 

Fix:
---
Don't delete implicit group(cluster_name) mapping of env.


Tests:
---
1. Create cluster and also add existing capacity
2. Modify existing capacity of env/stage having a cluster
3. Delete cluster and verify implicit group mapping is deleted
